BridgeAppSDK/SBAProfileItem.swift

```diff
@@ -121,6 +125,10 @@ extension SBAProfileItem {
         case SBAProfileTypeIdentifier.dictionary, SBAProfileTypeIdentifier.array:
             return (val as? SBAJSONObject)?.jsonObject() as? SBBJSONValue
             
+        case SBAProfileTypeIdentifier.set:
+            guard let set = val as? Set<AnyHashable> else { return nil }
+            return (Array(set) as SBAJSONObject).jsonObject() as? SBBJSONValue
```

Comment:
Note to self: Would be nice to merge the `SBAJSONObject` and `SBBJSONValue` protocols. ;)